### ISSUE TITLE
[BUG] - Minimum LO frequency in LF port mode

### ISSUE THREAD

**Issue by GlaserN** [zhinst/laboneq#38]
**Describe the bug**
The frequency of the SHFSG LO is restricted to a frequency minimum of 1GHz. When using the LF mode it should be possible to set the center frequency to lower frequencies, as this is also allowed in the L1 user interface.
This is however restricted in the controller:

```
line 666, in calc_outputs
    raise LabOneQException(
laboneq.core.exceptions.laboneq_exception.LabOneQException: Error on signal line 'F2/drive_line': (DeviceType.SHFSG) Local oscillator frequency 100000000.0 Hz is smaller than minimum 1000000000.0 Hz.
```

**Versions used:**
 - laboneq: 2.6.0

**Context**
- [ ] Do you know a workaround for the issue? If yes, please provide a short description. 
- [ ] Does the issue block your experiments?

**Comment by zi-2k** [comment]
This issue is fixed in 2.7.0.

**Comment by GlaserN** [comment]
Thanks for clarifying.
